src/test/java/net/wasdev/swagger/sample/fvt/test/ApplicationIT.java

```diff
@@ -10,24 +10,27 @@
 
 import org.junit.Test;
 
-public class ApplicationTest {
+public class ApplicationIT {
 	
 	@Test
 	public void testContacts(){
 		//check if endpoint works, should return empty JSON Array
+		System.out.println("***URL***: " + getResponse("contacts"));
```

Comment:
should these debug system.outs be removed or are they useful enough to leave in?